js/main.js

```diff
@@ -6,98 +6,105 @@ const combinationsList = document.querySelector(".comb-list");
 calculateButton.addEventListener("click", () => {
-    addCard();
-    clearInput();
-})
-
+  addCard();
+  // clearInput();
+});
 
 // Filter out sizes smaller than the min
 const filterOutTooSmall = (allCombinations, setMin) => {
-    filteredCombinations = []
-    for (let combination of allCombinations) {
-        let index = allCombinations.indexOf(combination);
-        console.log('index:' + index)
-        if (combination[1] >= setMin) {
-            filteredCombinations.push(combination);
-            console.log("I need to be removed:" + allCombinations[index]);
-        }
+  filteredCombinations = [];
+  for (let combination of allCombinations) {
+    let index = allCombinations.indexOf(combination);
+    console.log("index:" + index);
+    if (combination[1] >= setMin) {
+      filteredCombinations.push(combination);
+      console.log("I need to be removed:" + allCombinations[index]);
     }
-    console.log(filteredCombinations);
- 
-    return filteredCombinations;
-}
+  }
+  console.log(filteredCombinations);
+
+  return filteredCombinations;
+};
 
 // refactoring addCard
 const addCard = () => {
-    combinationsList.innerHTML = '';
-    let areaValue = parseFloat(area.value);
-    let setMinValue = parseFloat(setMin.value);
-    let setStepValue = parseFloat(setStep.value);
-    const combArray = areaCombinations(areaValue, setMinValue, setStepValue);
-    const filteredArray = filterOutTooSmall(combArray, setMinValue);
-    const strCombArray = formatCombinations(filteredArray);
-    createCard(strCombArray);
-    combinationsList.classList.remove('hide')
-}
+  combinationsList.innerHTML = "";
+  let areaValue = parseFloat(area.value);
+  let setMinValue = parseFloat(setMin.value);
+  let setStepValue = parseFloat(setStep.value);
+  const combArray = areaCombinations(areaValue, setMinValue, setStepValue);
+  const filteredArray = filterOutTooSmall(combArray, setMinValue);
+  const strCombArray = formatCombinations(filteredArray);
+  createCard(strCombArray);
+  combinationsList.classList.remove("hide");
+};
 
 // create card
-const createCard = strCombArray => {
-    for (let pair of strCombArray) {
-        let cardDiv = document.createElement('div');
-        cardDiv.classList.add("col");
-        cardDiv.classList.add("s6");
-        cardDiv.classList.add("m5");
-        cardDiv.innerHTML = `
-            <div class="card blue-grey darken-1">
+const createCard = (strCombArray) => {
+  for (let pair of strCombArray) {
+    let cardStage = document.createElement("div");
+    cardStage.classList.add("col");
+    cardStage.classList.add("s5");
+    // cardStage.classList.add("m5");
+
+    const dimensions = pair.split(" x ");
+
+    // Calculate stage size
+    // Multiply dimensions to get better results
+
+    cardStage.innerHTML = `
+            <p>${pair}</p>
+            <div class="card blue-grey darken-1"
+              style="width: ${dimensions[1] * 10}px; height: ${
+      dimensions[0] * 10
+    }px;">
```

Comment:
Sorry again for all the white-space changes – you can safely ignore then (but do go have a look at [Prettier](https://prettier.io/)).

The changes I've highlighted here are the main things we did.